Cython/Compiler/Parsing.py

```diff
@@ -1161,6 +1171,21 @@ def p_f_string_expr(s, unicode_value, pos, starting_index, is_raw):
         error(_f_string_error_pos(pos, unicode_value, starting_index),
               "empty expression not allowed in f-string")
 
+    if terminal_char == '=':
+        i += 1
+        while True:
+            if i >= size:
+                break  # error will be reported below
+
+            c = unicode_value[i]
+            if c.isspace():
+                i += 1
+                continue
+
+            terminal_char = c
+            expr_text = unicode_value[starting_index:i]
+            break
```

Comment:
Let's simplify this a little.
```suggestion
        while i < size and unicode_value[i].isspace():
            i += 1
        if i < size:
            expr_text = unicode_value[starting_index:i]
            terminal_char = unicode_value[i]
        # otherwise: error will be reported below
```